examples/analytics-iframe-transport.amp.html

```diff
@@ -13,7 +13,7 @@
 </head>
 <body>
   Here is some text above the ad.<br/>
-  <amp-ad width="300" height="400"
+  <amp-ad width="600" height="530"
```

Comment:
Ignore this - was changed in #10714 and will be rebased away once #10714 gets merged.